src/common/exif.cc

```diff
@@ -1065,7 +1065,10 @@ static bool _exif_decode_exif_data(dt_image_t *img, Exiv2::ExifData &exifData)
     if(FIND_EXIF_TAG("Exif.Photo.UserComment"))
     {
       std::string str = pos->print(&exifData);
-      dt_metadata_set_import(img->id, "Xmp.dc.description", str.c_str());
+      Exiv2::CommentValue value(str);
+      std::string comment = value.comment();
```

Comment:
additionally you can use string copy constructor which should be tad faster ;)

```cpp
std::string comment(value.comment());
```

Reply:
Well, no way, I have to learn to understand properly... (any advice for learning basics of C++ BTW ?...)
Thanks !

Reply:
@johnny-bit, these 2 initialization methods both call copy constructor and are totally identical. I think assignment way to write it is more readable.

Reply:
don't sweat it ;) C++ is far more complex (imo) than C, has it's own advantages and disadvantages... I think best reading material is "Effective (Modern) C++" and "Effective STL" and maybe some others. Most of teh time knowledge of C is enough to write good enough C++ :)

Reply:
>  I think best reading material is "Effective (Modern) C++" and "Effective STL"

Thanks. I'll make a try on this (if the neurons that I have left support it) !

Reply:
As identical, I let the code as it is...